python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -161,11 +160,14 @@ def __init__(self, function_spec, input_args_with_spec,
             input_args_with_spec(list[InputSpec]): actual input args with some arguments replaced by InputSpec.
             input_kwargs_with_spec(list[{string:InputSpec}]): actual input kwargs with some arguments replaced by InputSpec.
             class_instance(object): a instance of class `Layer`.
+            **kwargs(dict): manage other arguments used for better scalability
```

Comment:
LGTM, but in the future, can we think a way to manage the kwargs that users/engineers can pass to the interface. Code document or something else? I haven't thought about it, but I think it can be helpful to let caller know what kwargs they can pass.

Reply:
Thanks, it's a nice advice.